generate_nibetaseries_images.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+# Generate Dockerfile.
+generate_docker() {
+  docker run --rm kaczmarj/neurodocker:0.5.0 generate docker \
+    --base=neurodebian:stretch-non-free \
+    --install gcc g++ graphviz \
+    --pkg-manager=apt \
+    --user=neuro \
+    --miniconda \
+      conda_install="python=3.6" \
+      pip_install="jupyter seaborn tox" \
+      create_env="neuro_py36" \
+      activate=true \
+    --copy . /src/nibetaseries \
+    --user=root \
+    --run 'chown -R neuro:users /src/nibetaseries' \
```

Comment:
I used `chown` instead of `chmod` since it constrains the usage of the container to the user `neuro` (or `root`). Doesn't make a big difference in the functionality, but perhaps increases security of the container?? (also not something I'm overly worried about, currently).

Reply:
Fair point. I'm with you on that.